pandas/_libs/algos_common_helper.pxi.in

```diff
@@ -587,7 +587,7 @@ def get_dispatch(dtypes):
 
 {{for name, c_type, dtype in get_dispatch(dtypes)}}
 
-cpdef ensure_{{name}}(object arr, copy=True):
+def ensure_{{name}}(object arr, copy=True):
```

Comment:
are these for sure not called in cython code?

Reply:
non-cimported only